iroha_network/src/lib.rs

```diff
@@ -7,7 +7,7 @@ use async_std::{
     sync::RwLock,
 };
 use iroha_derive::{log, Io};
-use iroha_error::{Error, Result};
+use iroha_error::{error, Error, Result};
```

Comment:
Do you need to import `Error` here, it seems it is not used. If you need to import it because of the error macro, it is better to use full path there like `iroha_error::Error::msg(..)`. This is the place where I think it is necessary to make exception for our imports style guide, so it will help with user experience.